src/LibSDL2.jl

```diff
@@ -219,7 +262,7 @@ mutable struct SDL_BlitMap end
 
 struct SDL_Surface
     flags::Uint32
-    format::Ptr{Cvoid} # format::Ptr{SDL_PixelFormat}
+    format::Ptr{SDL_PixelFormat}
```

Comment:
Should this stay a Cvoid?

Reply:
No that's ok, Clang.jl probably just didn't have full support for pointer fields before. You can see from the comment that it did recognize the type of the field but didn't use it for some reason.